ci/do_ci.sh

```diff
@@ -393,6 +393,16 @@ elif [[ "$1" == "cmake.exporter.otprotocol.with_async_export.test" ]]; then
   make -j $(nproc)
   cd exporters/otlp && make test
   exit 0
+elif [[ "$1" == "cmake.w3c.trace-context.build-server" ]]; then
+  echo "Building w3c trace context test server"
+  cd "${BUILD_DIR}"
+  rm -rf *
+  cmake "${CMAKE_OPTIONS[@]}"  \
+          -DBUILD_W3CTRACECONTEXT_TEST=ON \
+          -DCMAKE_CXX_STANDARD=${CXX_STANDARD} \
+          "${SRC_DIR}"
+  eval "$MAKE_COMMAND"
```

Comment:
nit, `make -j $(nproc)` to utilize all available cores to build.

Reply:
$MAKE_COMMAND should expand to 
```
make -k -j  \$(nproc)
```

I am not quite sure about what `\` before $(nproc) does, but I imagine it would utilize all cores ?